operator-sdk requires Go 1.12

**Describe the bug**
Metal3 deployment fails due to github.com/operator-framework/operator-sdk requiring newer Go version

**To Reproduce**
Attempt to run `make` in master branch

**Observed behavior**
```
Syncing github.com/operator-framework/operator-sdk
+ '[' '!' -d /home/jenkins-build/go/src/github.com/operator-framework/operator-sdk ']'
+ pushd /home/jenkins-build/go/src/github.com/operator-framework/operator-sdk
~/go/src/github.com/operator-framework/operator-sdk ~/build/workspace/ocs4-metal3/53/dev-scripts
+ git am --abort
Resolve operation not in progress, we are not resuming.
+ true
+ git checkout master
Already on 'master'
+ git fetch origin
+ git rebase origin/master
Current branch master is up to date.
+ test 2 -gt 2
+ popd
~/build/workspace/ocs4-metal3/53/dev-scripts
+ pushd /home/jenkins-build/go/src/github.com/operator-framework/operator-sdk
~/go/src/github.com/operator-framework/operator-sdk ~/build/workspace/ocs4-metal3/53/dev-scripts
+ git checkout master
Already on 'master'
+ make install
make[1]: Entering directory `/home/jenkins-build/go/src/github.com/operator-framework/operator-sdk'
go build github.com/operator-framework/operator-sdk/version: module requires Go 1.12
go build github.com/google/gofuzz: module requires Go 1.12
go build github.com/operator-framework/operator-sdk/internal/util/operator-registry: module requires Go 1.12
go build github.com/pelletier/go-toml: module requires Go 1.12
go build github.com/operator-framework/operator-sdk/internal/util/diffutil: module requires Go 1.12
go build sigs.k8s.io/controller-tools/pkg/internal/general: module requires Go 1.12
go build sigs.k8s.io/controller-tools/pkg/crd/util: module requires Go 1.12
go build golang.org/x/sys/unix: module requires Go 1.12
go build github.com/operator-framework/operator-sdk/internal/flags/watch: module requires Go 1.12
go build github.com/operator-framework/operator-sdk/pkg/helm/watches: module requires Go 1.12
go build github.com/operator-framework/operator-sdk/cmd/operator-sdk/completion: module requires Go 1.12
go build sigs.k8s.io/controller-tools/pkg/util: module requires Go 1.12
make[1]: *** [install] Error 1
make[1]: Leaving directory `/home/jenkins-build/go/src/github.com/operator-framework/operator-sdk'
make: *** [repo_sync] Error 2
```

--- a/01_install_requirements.sh
+++ b/01_install_requirements.sh
@@ -177,3 +177,9 @@ if [ ! -f ${oc_tools_dir}/${oc_tools_local_file} ] || [ $oc_date -lt 1559308936 
   tar xvzf ${oc_tools_local_file}
   sudo cp oc /usr/local/bin/
 fi
+
+# Install operator-sdk
+if ! which operator-sdk 2>&1 >/dev/null ; then
+    sudo wget https://github.com/operator-framework/operator-sdk/releases/download/v0.9.0/operator-sdk-v0.9.0-x86_64-linux-gnu -O /usr/local/bin/operator-sdk
+    sudo chmod 755 /usr/local/bin/operator-sdk
+fi

--- a/03_ocp_repo_sync.sh
+++ b/03_ocp_repo_sync.sh
@@ -25,14 +25,5 @@ mkdir -p $GOPATH/bin
 curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 export PATH="${GOPATH}/bin:$PATH"
 
-# Install operator-sdk for use by the baremetal-operator
-sync_repo_and_patch github.com/operator-framework/operator-sdk https://github.com/operator-framework/operator-sdk.git
-
-# Build operator-sdk
-pushd "${GOPATH}/src/github.com/operator-framework/operator-sdk"
-git checkout master
-make install
-popd
-
 # Install baremetal-operator
 sync_repo_and_patch github.com/metal3-io/baremetal-operator https://github.com/metal3-io/baremetal-operator.git

